### PR TITLE
feat: add examples of s3 URL

### PR DESCRIPTION
**Pull request by WenyXu**

### CODE REVIEW

**Review by nicecui — COMMENTED** [review]
👍 Just need a little change

File: docs/reference/sql/copy.md [hunk 1]
```diff
@@ -64,10 +64,28 @@ COPY tbl FROM '/path/to/folder/xxx.parquet' WITH (FORMAT = 'parquet');
...
-`COPY tbl FROM 's3://path/to/xxx.parquet' WITH (FORMAT = 'parquet') CONNECTION(BUCKET = 'us-west-2');`. 
+COPY tbl FROM 's3://bucket/path/to/xxx.parquet' WITH (FORMAT = 'parquet') CONNECTION(BUCKET = 'us-west-2');. 
+```
+
+**URL**
+
+Notes: You should specify a bucket using `S3://bucket`. The following example shows the correct format.
+
+```
+S3://bucket-name/key-name
```
[the remark this answers is not in the document]
Comment (WenyXu) [reply docs/reference/sql/copy.md:77]: Yep, `Copy tbl from '{URL}' ...`

File: docs/reference/sql/copy.md [hunk 2]
```diff
@@ -64,10 +64,28 @@ COPY tbl FROM '/path/to/folder/xxx.parquet' WITH (FORMAT = 'parquet');
 #### S3
 
 When you try to copy data from(to) S3:
+
 ```sql
-`COPY tbl FROM 's3://path/to/xxx.parquet' WITH (FORMAT = 'parquet') CONNECTION(BUCKET = 'us-west-2');`. 
+COPY tbl FROM 's3://bucket/path/to/xxx.parquet' WITH (FORMAT = 'parquet') CONNECTION(BUCKET = 'us-west-2');.
```
[the remark this answers is not in the document]
Comment (WenyXu) [reply docs/reference/sql/copy.md:69]: Let's unify it to S3

**Comment by WenyXu** [comment]
PTAL @nicecui

File: docs/reference/sql/copy.md [hunk 3]
```diff
@@ -64,10 +64,28 @@ COPY tbl FROM '/path/to/folder/xxx.parquet' WITH (FORMAT = 'parquet');
 #### S3
 
 When you try to copy data from(to) S3:
+
 ```sql
-`COPY tbl FROM 's3://path/to/xxx.parquet' WITH (FORMAT = 'parquet') CONNECTION(BUCKET = 'us-west-2');`. 
+COPY tbl FROM 'S3://bucket/path/to/xxx.parquet' WITH (FORMAT = 'parquet') CONNECTION(BUCKET = 'us-west-2');.
```
Comment (nicecui) [inline docs/reference/sql/copy.md:69]:
```suggestion
COPY tbl FROM '<url>' WITH (FORMAT = 'parquet') CONNECTION(BUCKET = 'us-west-2');. 
```

File: docs/reference/sql/copy.md [hunk 4]
```diff
@@ -64,10 +64,28 @@ COPY tbl FROM '/path/to/folder/xxx.parquet' WITH (FORMAT = 'parquet');
...
 When you try to copy data from(to) S3:
+
 ```sql
-`COPY tbl FROM 's3://path/to/xxx.parquet' WITH (FORMAT = 'parquet') CONNECTION(BUCKET = 'us-west-2');`. 
+COPY tbl FROM 'S3://bucket/path/to/xxx.parquet' WITH (FORMAT = 'parquet') CONNECTION(BUCKET = 'us-west-2');. 
+```
+
+**URL**
+
+Notes: You should specify a bucket using `S3://bucket`. The following example shows the correct format.
```
Comment (nicecui) [inline docs/reference/sql/copy.md:74]:
```suggestion
Notes: You should specify a file using `S3://bucket/key-name`. The following example shows the correct format.
```

File: docs/reference/sql/copy.md [hunk 5]
```diff
@@ -64,10 +64,28 @@ COPY tbl FROM '/path/to/folder/xxx.parquet' WITH (FORMAT = 'parquet');
...
+
+```
+S3://bucket-name/key-name
```
Comment (nicecui) [inline docs/reference/sql/copy.md:77]:
```suggestion
S3://my-bucket/data.parquet
```

**Comment by WenyXu** [comment]
PTAL @nicecui

**Review by nicecui — APPROVED** [review]
👍

**Review by xtang — APPROVED** [review]

**Pull request merged**